### ISSUE TITLE
On Loss Functions for Deep Neural Networks in Classification

### ISSUE THREAD

**Issue by Wanwannodao** [Wanwannodao/DeepLearning#3]
[On Loss Functions for Deep Neural Networks in Classification](https://arxiv.org/abs/1702.05659?utm_content=bufferd6636&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer)
